src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -263,6 +264,14 @@ public void configure(Context context) {
                     + "properly work!");
         } // if else
         
+        String csvSeparator = context.getString("csv_separator", ",");        
+        if (csvSeparator != null && csvSeparator.length() > 0) {
```

Comment:
These lines are not necessary sicne there is always a value for "csv_separator".